gravitee-apim-console-webui/src/management/integrations/integrations.component.html

```diff
@@ -0,0 +1,113 @@
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<div>
+  <div class="page-header">
+    <h1 class="page-header__page-title">Integrations</h1>
+    <span class="page-header__description"
+      >Connect to third-party API gateways and event brokers to create a unified control plane and API portal with Gravitee</span
+    >
+  </div>
+  <mat-card>
+    <mat-card-content>
+      <div class="card-header">
+        <div class="card-header__title">
+          <h3>Integrations</h3>
+        </div>
+        <div class="card-header__actions">
+          <button
+            mat-raised-button
+            color="primary"
+            [disabled]="isLoading"
+            data-testid="create-integration-button"
+            routerLink="create-integration"
+          >
+            Create Integration
+          </button>
+        </div>
+      </div>
+
+      <ng-template #noIntegrations>
+        <div class="no-integrations">
+          <div class="no-integrations__img">
+            <img class="banner" src="/assets/integrations-banner.png" alt="Integrations banner" />
+          </div>
+          <div class="no-integrations__message">
+            <h2 class="header">No integrations yet</h2>
+            <p class="description">Create an integration to start importing APIs and event streams from a 3rd-party provider.</p>
+          </div>
+        </div>
+      </ng-template>
+    </mat-card-content>
+
+    <ng-container *ngIf="integrations.length || isLoading; else noIntegrations">
+      <gio-table-wrapper
+        [disableSearchInput]="true"
+        [length]="nbTotalInstances"
+        [filters]="filters"
+        (filtersChange)="onFiltersChanged($event)"
+      >
+        <table mat-table [dataSource]="integrations" aria-label="Integrations table">
+          <caption style="display: none">
+            Table with Integrations
+          </caption>
+          <ng-container matColumnDef="name">
+            <th mat-header-cell *matHeaderCellDef id="name">Name</th>
+            <td mat-cell *matCellDef="let integration">{{ integration.name }}</td>
+          </ng-container>
+
+          <ng-container matColumnDef="owner">
+            <th mat-header-cell *matHeaderCellDef id="owner">Owner</th>
+            <!-- toDo: remove Not implemented string when Backend Ready:           -->
+            <td mat-cell *matCellDef="let integration">{{ integration.owner || 'Not Implemented' }}</td>
```

Comment:
```suggestion
            <td mat-cell *matCellDef="let integration">{{ integration.owner || '' }}</td>
```

to have a better UI the time we introduce the owner thing 🙈